### ISSUE TITLE
Instructions for use by non-savvy people

### ISSUE THREAD

**Issue by rbenson** [ThePacielloGroup/CCAe#200]
**Is your feature request related to a problem? Please describe.**
We are hoping to promote CCA in our organization. While CCA is meant to be intuitive, leadership is asking for instructions on use. Can you guys throw an one pager that covers use or add it to the app itself?

**Comment by ScreenReaderTester** [comment]
I agree that it should be intuitive but is not and instructions would be very helpful.  Some people using the tool just click on any pixel.

There is a training video how to use CCA at
https://www.section508.gov/test/web-software/andi-training-videos/color-contrast-analyzer
At 59 seconds into the video, it says "Select the foreground color picker button and move the color picker box over the darkest part of the text and select enter".

https://webaccess.msu.edu/Tutorials/color-contrast.html#contrast has instructions for older version of the tool.
"There are several versions available and we currently recommend the older versions because they more easily allow pixel specific selection which is essential when validating the contrast of fonts against a background."

<img width="300" alt="Letter P" src="https://user-images.githubusercontent.com/47258588/121214331-03663900-c84d-11eb-821b-1c4d79a3b59d.png">
